library/src/main/java/com/pokegoapi/api/inventory/Inventories.java

```diff
@@ -33,13 +33,13 @@
 import com.pokegoapi.exceptions.LoginFailedException;
 import com.pokegoapi.exceptions.RemoteServerException;
 import com.pokegoapi.main.ServerRequest;
+import com.pokegoapi.api.player.Stats;
 
 import lombok.Getter;
 
 import java.util.ArrayList;
 import java.util.List;
 
-
 public class Inventories {
```

Comment:
There is no commited change in line 150 on the class Inventories.java in this pr.